src/github.rs

```diff
@@ -26,14 +46,14 @@ actix_web::HttpResponse::Found()
 }
 #[derive(serde::Deserialize)]
 pub struct RepoParams {
```

Comment:
These are technically inputs to Identity, so RepoParams does not make much sense. Maybe call this `IdentityInput`.